### DISCUSSION TITLE
I'm lost - how to setup my accounts?

### DISCUSSION THREAD

**Discussion by markusdd** [the-djmaze/snappymail#1584] [Q&A]
Hi,
I'm very interested in this project and got it running using the docker compose sqlite example.
container is up, I can login into the admin interface, I can reach the main login screen behind my traefik, ACME also works.
So far so good, piece of cake.

But now: what do I actually have to do ? My usecase is actually simple: I just want to add my one gmx.com and 2 gmail accounts
and have them in one view (common inbox) as a webmail client. No local mailserver or own mail domain involved or needed.

what i did in the domain view I added my accounts without any wildcards and when I run test IMAP, SMTP etc is green, so that works.

But the domain testbox says 'no domain match' when I use this exact login?  Also: when I then try to login in the user view using my gmx account it just gives me some info it cannot connect to tcp localhost:143, which I would expect, because that is not the mailserver it should contact.

Either I have some fundamental misunderstanding how that whole thing works, or I simply did not find the relevant tutorial. I went to the wiki multiple times and I've come up empty.

In essence: HELP! :sweat_smile:

**Comment by markusdd** [comment]
So I have this setup: 
![image](https://github.com/the-djmaze/snappymail/assets/25175069/3192eee9-f6f9-40b1-bc95-f5aa67379fba)

And I have my 2 addresses in the whitelist:
![image](https://github.com/the-djmaze/snappymail/assets/25175069/3afd91e3-5e83-459e-bd98-d845a54d5a88)

When I then try to test the domain and use one oof these logins, it works.

But when I then use the domain test button in the overview, it says 'no domain match'. Login in the user view also does not work.
I'm puzzled.

**Comment by markusdd** [comment]
Ok, meanwhile I found out that my gmx account now works, but I am completely stuck with gmail.

With app-password approach it tells me my account dooes already exist, with the second one, that does not have 2FA turned on for testing, it says invalid credentials although it works as a testing account in the admin interface.

so...I have also created an oath credential pair because I had then setup for my home assistant anyway, but what is the redirect URI? I can click on oath in the login screen for gmail but it errors out with code 400, redirect URI invalid.

It surely isn't the /demo one from the wiki is it?

EDIT:

From searching the coode I found /?LoginGMail as an attachment which indeed them brings me top the oauth page of google, but once I have completed it I am back at the login screen. Also: How does this work if I am logged in with my gmx.com account and want to add the 2 gmails to the same view, there I am only getting the normal password prompt, which then also does not work.

**Comment by the-djmaze** [comment]
LoginGMail is the gmail extension using OAuth2.
Currently it only works as login and not as additional account.
